rpc_jobs/rpc_openstack.yml

```diff
@@ -196,6 +196,31 @@
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
       - 'PM-push-trigger_{repo_name}-{branch}-{image}-{scenario}-{action}'
 
+- project:
+    name: "rpc-openstack-newton-mnaio-postmerge"
+    repo_name: "rpc-openstack"
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branch: "newton"
+    jira_project_key: "RO"
+    image:
+      - xenial_mnaio:
+          IMAGE: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
+          FLAVOR: "onmetal-io1"
+          REGIONS: "IAD"
+          FALLBACK_REGIONS: ""
+      - xenial_mnaio_no_artifacts:
+          IMAGE: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
+          FLAVOR: "onmetal-io1"
+          REGIONS: "IAD"
+          FALLBACK_REGIONS: ""
+    scenario:
+      - "ironic"
+      - "swift"
+    action:
+      - deploy
```

Comment:
To reduce some duplication, you could optionally have the common settings for all images in the deploy action instead of under each image. eg:

```
    image:
      - xenial_mnaio
      - xenial_mnaio_no_artifacts
    scenario:
      - "ironic"
      - "swift"
    action:
      - deploy:
          IMAGE: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
          FLAVOR: "onmetal-io1"
          REGIONS: "IAD"
          FALLBACK_REGIONS: ""
```